### PR TITLE
Build: Exclude stories from collecting coverage

### PR DESCRIPTION
**Pull request by mbelsky**
## What I did
I've excluded story files from collecting coverage.

## How to test

This pr's [`ci/circleci: coverage`](https://codecov.io/gh/storybookjs/storybook/tree/b55d7c386bacb466cf5c6c3c83708b083e910e8f) greater than  storybookjs:next's [`ci/circleci: coverage`](https://codecov.io/gh/storybookjs/storybook/tree/d28d8b598dfbfa4ff2e361e75937d89f83566dd1)

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -61,6 +61,7 @@ module.exports = {
     'app/**/*.{js,jsx,ts,tsx}',
     'lib/**/*.{js,jsx,ts,tsx}',
     'addons/**/*.{js,jsx,ts,tsx}',
+    '!**/*.stories.{js,jsx,ts,tsx}',
```
Comment (mbelsky) [inline jest.config.js:64]: I've tried to put this pattern in `coveragePathIgnorePatterns`, but it fails with `SyntaxError: Invalid regular expression: /**/*.stories.{js,jsx,ts,tsx}/: Nothing to repeat`.

Comment (shilman) [reply jest.config.js:64]: @mbelsky if it helps, i think `**` is a glob not a regex

Comment (mbelsky) [reply jest.config.js]: Fixed, thanks

**Review by shilman — APPROVED** [review]
Nice one! 🎉

**Pull request merged**